### DISCUSSION TITLE
More Stat Options

### DISCUSSION THREAD

**Discussion by Jamacaw** [gameserverapp/Platform#249] [Suggestions]
One of my favorite things on the GSA dashboard is the stats it provides for daily unique players, characters created, hours played, etc.  Would be really awesome to be able to have options to filter these stats for a certain period of time, get more than a weeks view, etc. 

It would be great as well if there were some more things tracked such as daily peak online users, game related stats, achievement related stats.

Options on the dashboard to show/hide different graphs would be useful as well, especially if more stats are added.

**Comment by mvaessen** [comment]
I think this suggestion is a duplicate of #208. Would you mind adding your idea's to that suggestion?

With the customisable dashboards you can create multiple dashboards, with customisable graphs.
